### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sfc-gh-halu**
Clarify as customer has question about `target_uri`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,11 +25,19 @@ create_session(connection_parameters)
...
+
 #### SnowSQL Configuration file
 [SnowSQL Configuration file](https://docs.snowflake.com/en/user-guide/snowsql-config.html) is a familiar concept among existing SnowSQL CLI users and a neccessary way to establish connection to Snowflake if you intend to use MLflow CLI for model deployment.
 For the Snowflake deployment plugin, the `target_uri` needs to have the`snowflake` scheme.
 Connection parameters can be specified by adding `?connection={CONNECTION_NAME}`.
 The `CONNECTION_NAME` references the connection specified in the SnowSQL configuration file e.g. `snowflake:/?connection=connections.ml`.
+
+#### Specification for target_uri
+If you are using MLflow python API and already have created Snowflake session, then `target_uri` is `snowflake` which will ensure MLflow choose Snowflake as target platform.
+If you are using MLflow CLI, then you need to specify the connection name based on your SnowSQL configuration file. In this case, your `target_uri` would be `snowflake:/?connection={connection_name}`. The library would take care creating the Snowflake session for you.
```
Comment (sfc-gh-sdas) [inline README.md:38]: can we create a simple util library to help creating the `target_uri` from a dict of connection parameters?

Comment (sfc-gh-halu) [reply README.md:38]: `target_uri` is independent from connection parameters. it's most likely to be literal `snowflake` for majority of the customers.

Comment (sfc-gh-sdas) [reply README.md:38]: For some reason, this paragraph is not making it clear.

**Review by sfc-gh-sdas — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sfc-gh-amauser) [inline README.md:38]:
```suggestion

#### Specification for target_uri
If you are using the MLflow python API and already have created a Snowflake session, then `target_uri` is `snowflake`. 
If you are using the MLflow CLI, you need to specify the connection name based on your SnowSQL configuration file. In this case, your `target_uri` would be `snowflake:/?connection={connection_name}`. The library would take care of creating the Snowflake session for you.
```

Comment (sfc-gh-halu) [reply README.md:38]: It's the same as above passage, try to rephrase more explicitly since got a question from one customer.

Comment (sfc-gh-amauser) [reply README.md]: I think it would be better to extend the first passage then instead of near-duplicating it.

**Pull request merged**